locksmith/src/controllers/metadataController.ts

```diff
@@ -2,16 +2,15 @@
 /* eslint-disable no-use-before-define */
 import { Response } from 'express-serve-static-core' // eslint-disable-line no-unused-vars, import/no-unresolved
 import Normalizer from '../utils/normalizer'
-import { SignedRequest } from '../types' // eslint-disable-line no-unused-vars, import/no-unresolved
```

Comment:
Any reason to remove this typedef?

Reply:
Cutting over importing the controller in the router revealed a type mismatch that previously wasn't raised. While the types in this controller are as expected, the path mapping code in the router expects an object that contains a `Request` making this incompatible